extensions/blocks/map/edit.js

```diff
@@ -32,6 +32,7 @@ import Locations from './locations';
 import Map from './component.js';
 import MapThemePicker from './map-theme-picker';
 import { settings } from './settings.js';
+import previewPlaceholder from './map-preview.png';
```

Comment:
We'll most likely need to switch to an optimized jpg file, to match the other block example images.

Reply:
Fixed - now a JPG at 512 pixel resolution